### ISSUE TITLE
How can we know which is current slide is? 

### ISSUE THREAD

**Issue by v3ss0n** [CoachLogix/ember-paper-swiper#1]
By material design spec button to first slides need to be - Next - Next.

**Comment by miguelcobain** [comment]
Can you provide a link?

**Comment by v3ss0n** [comment]
https://material.io/guidelines/growth-communications/onboarding.html#onboarding-quickstart

**Comment by v3ss0n** [comment]
>Display a “Get Started” button throughout the animation, and loop through the animation continuously until the “Get Started” action is tapped.

**Comment by v3ss0n** [comment]
Sorry i am wrong. But would like to implement First / Second buttons as Next, buttons. 
I am trying to access isFirst/isLast variables.

**Comment by v3ss0n** [comment]
```
{{#paper-swiper class="" fullscreen="false" as |swiper|}}

    {{#swiper.callout}}
    {{log swiper.currentSlide}}
    {{log swiper}}
    {{#if swiper.isLast}}
      <a href="#" class="login-link layout-align-center-center  layout-row" 
      {{action 'signInToFacebook'}}>
        {{#paper-button raised=true primary=true class='flex fb-login force-font-myanmar-onboarding'}}
          {{#if session.isAuthenticated}}
            {{t "application.login.nowlogin"}}
          {{else}}
            {{t "application.login.facebooklogin"}}
          {{/if}}
        {{/paper-button}}
      </a>
    {{else}}
       {{next-button}}
    {{/if}}
    {{/swiper.callout}}

    {{#swiper.slide class="bear-slide" as |slide|}}
      {{slide.media src="assets/images/doctor.png" class="sszz"}}
      {{#slide.headline class=""}}

      {{/slide.headline}}
      {{#slide.subhead class=""}}

      {{/slide.subhead}}
    {{/swiper.slide}}
```

**Comment by miguelcobain** [comment]
Oh, I see. You want to display different buttons on each slide. It should be possible to do something like that. I'll investigate.

It is, however not parte of the spec. The button is always the same.

**Comment by miguelcobain** [comment]
I published version `0.0.3`. You can now use `swiper.currentSlide` to get the index of the current slide.

**Comment by v3ss0n** [comment]
thanks a lot!
